src/settings/defaults.py

```diff
@@ -275,8 +275,23 @@ def __setattr__(self,k,v):
     d.USEPRAKTOMATTESTER = False
  
     # Alternatively: Run everything in a docker instance, to provide higher
-    # insulation. Should not be used together iwth USEPRAKTOMATTESTER.
+    # insulation. Should not be used together with USEPRAKTOMATTESTER.
+    
+    # It is recomendet to use DOCKER and not a tester account
```

Comment:
d

Reply:
@physikerwelt  (just recognised your comment from may 2017.) What do you mean with "d" ?  
"recomendet" => "recomended"  or what else?

Reply:
Yes. recomended